modules/op-release-notes-1-8.adoc

```diff
@@ -22,19 +22,21 @@ include::snippets/technology-preview.adoc[]
 
 
 [id="deprecated-features-1-8_{context}"]
-== Deprecated features
+== Deprecated and removed features
 
-* Support for the `tekton.dev/v1alpha1` API version, which was deprecated in {pipelines-title} GA 1.6, is planned to be removed in the upcoming {pipelines-title} GA 1.9 release.
+* In {pipelines-title} 1.8, support for the `tekton.dev/v1alpha1` API version, which was deprecated in {pipelines-title} GA 1.6, is planned to be removed in the upcoming {pipelines-title} GA 1.9 release.
```

Comment:
@vdemeester this sentence mentions both 1.8 and 1.9. Can you clarify if only `Condition` and `PipelineResource` are being removed in 1.8 or all `v1alpha1` (are there any left?)?

Reply:
- The change on line 27 is purely a language issue: Our standard for deprecation and removal release notes asks us to open with `In <product/component name> <current version>,` This is why we mention 1.8, above.  
- The rest of Pavol's question about  `Condition` and `PipelineResource` stands.
- However, I don't have a task about deprecating `Condition` and I don't see it mentioned anywhere. Should we have a doc task for `Condition`?

Reply:
@ppitonak good question. *All* `v1alpha` object that are available in `v1beta1` will be removed. `PipelineResource` and `Condition` as well. But, there will be still some CRDs in `v1alpha1` (the `Run` object that will probably be renamed `CustomRun` is probably gonna be `v1alpha1` for a little while)

Reply:
@vdemeester @ppitonak I've added a release note item for `Condition` one line down from the one for `PipelineResource`. I'm listing them separately because it seems like `Condition` was never named as a Tech Preview item...

Reply:
If I understand what you're saying, @vdemeester:
- All `v1alpha` objects that available in `v1beta` are being removed **in 1.8**.
- All remaining `v1alpha` objects will be removed in 1.9.

Reply:
> All remaining v1alpha objects will be removed in 1.9.
It's safer to just claim that it will be removed in *some* future release

Reply:
@rolfedh Sorry I missed this. I agree that we should avoid making promises about specific future releases. Here are the guidelines from the SSG: https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future

Reply:
All remaining `v1alpha1` object are "always" subject to be removed in any future released any way (that's the reason why they are alpha 😅). For example `v1alpha1.Run` is most likely going to be renamed `v1alpha1.CustomRun` (or `v1beta1.CustomRun`).